force_gromacs/io/gromacs_coordinate_reader.py

```diff
@@ -0,0 +1,182 @@
+import logging
+import re
+
+import numpy as np
+
+from .base_file_reader import BaseFileReader
+
+log = logging.getLogger(__name__)
+
+
+class GromacsCoordinateReader(BaseFileReader):
```

Comment:
Basic coordinate file parser, returning a dictionary containing information on molecule and atoms types, as well as simulation coordinate and cell dimensions for each frame. The form of the return value from `read` could be improved upon to a more regular format later on.